internal/resources/google/compute_cost_component_helpers.go

```diff
@@ -64,6 +67,99 @@ func computeCostComponent(region, machineType string, purchaseOption string, ins
 	}
 }
 
+// customComputeCostComponent returns a cost component for custom Compute instance usage.
+func customComputeCostComponents(region, machineType string, purchaseOption string, instanceCount int64, monthlyHours *float64) []*schema.CostComponent {
+	sustainedUseDiscount := 0.0
+	fixPurchaseOption := "Preemptible"
+	if strings.ToLower(purchaseOption) == "on_demand" {
+		fixPurchaseOption = "OnDemand"
+		switch strings.ToLower(strings.Split(machineType, "-")[0]) {
+		case "c2", "n2", "n2d":
+			sustainedUseDiscount = 0.2
+		case "n1", "f1", "g1", "m1":
+			sustainedUseDiscount = 0.3
+		}
+	}
+
+	var re = regexp.MustCompile(`(\D.+)-(\d+)-(\d.+)`)
+	firstMachineTypeInfo := re.ReplaceAllString(machineType, "$1")
+	strCPUAmount := re.ReplaceAllString(machineType, "$2")
+	strRAMAmount := re.ReplaceAllString(machineType, "$3")
+
+	instanceType := ""
+	if firstMachineTypeInfo != "custom" {
+		instanceType = strings.ToUpper(strings.Split(firstMachineTypeInfo, "-")[0])
+	}
+
+	ext := ""
+	if strings.Contains(strRAMAmount, "ext") {
+		ext = "Extended"
+		strRAMAmount = strings.Split(strRAMAmount, "-")[0]
+	}
+
+	numberOfCores, err := strconv.ParseInt(strCPUAmount, 10, 64)
+	if err != nil {
+		log.Warnf("Could not parse the custom number of Cores for %s", machineType)
+		return nil
+	}
+
+	mbOfRAM, err := strconv.ParseInt(strRAMAmount, 10, 64)
+	if err != nil {
+		log.Warnf("Could not parse the custom amount of Ram for %s", machineType)
+		return nil
+	}
+
+	qty := decimal.NewFromFloat(730)
```

Comment:
```suggestion
	qty := decimal.NewFromFloat(schema.HourToMonthUnitMultiplier)
```
Realise this was probably just copied from how it previously was though.